web/src/engine/websites/XCaliBRScans.ts

```diff
@@ -19,36 +58,106 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
-}
 
-// Original Source
-/*
-class XCaliBRScans extends WordPressMangastream {
-
-    constructor() {
-        super();
-        super.id = 'xcalibrscans';
-        super.label = 'xCaliBR Scans';
-        this.tags = [ 'webtoon', 'english' ];
-        this.url = 'https://xcalibrscans.com';
-        this.path = '/manga/list-mode/';
-        this.requestOptions.headers.set('x-referer', this.url);
-
-        this.config = {
-            throttle: {
-                label: 'Throttle Requests [ms]',
-                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests.\nThe website may block images for to many consecuitive requests.',
-                input: 'numeric',
-                min: 1000,
-                max: 7500,
-                value: 1500
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+
+        //There are 3 different type of chapters scrambling
+        // Nothing : https://xcalibrscans.com/the-first-ancestor-in-history-chapter-1/
+        // div#readerarea p img
+        // 0
+        // flipped single :  https://xcalibrscans.com/the-first-ancestor-in-history-chapter-86/
+        // div#readerarea div.kage img
+        // 1
+        // flipped and splitted : https://xcalibrscans.com/above-ten-thousand-people-chapter-175/
+        // div#readerarea div.kage div.sword_box div.sword img
+        // 2
+
+        const uri = new URL(chapter.Identifier, this.URI);
+        const request = new FetchRequest(uri.href);
+        const data = await FetchWindowScript<pageScriptResult>(request, pagescript,2500);
+        // HACK: bypass 'i0.wp.com' image CDN to ensure original images are loaded directly from host
```

Comment:
No need for this hack, there is a transformer providing the `DeProxify` method to cleanup image links